R/google_drive.R

```diff
@@ -3,7 +3,9 @@
 #' @param teamDriveId - in case you want to search for team drive
 #' @param path - This should be ID of the folder since searching with folder name doesn't always work as expected.
 #' @param type - object type that you want to include in your query result.
-listItemsInGoogleDrive <- function(teamDriveId = NULL, path = NULL, type =  c("csv", "tsv", "txt", "folder", "xls", "xlsx")){
+#' @param n_max - number of max items to return
+#' @param search - query string
```

Comment:
It should be "pattern" instead of "search". Also add that the condition of the recursive saerch.

Reply:
Updated as suggested.